lib/logger/index.js

```diff
@@ -30,4 +32,27 @@ const log = new Logger({
 
 log.addStream({ type: 'raw', stream: logstashStream })
 
-global.log = log
+const logProxy ={};
+
+logProxy.trace = (...args) =>{
+    log.trace(...args, {requestId: namespace.get('requestId')})
+}
+
+logProxy.debug = (...args) =>{
+    log.debug(...args, {requestId: namespace.get('requestId')})
+}
+logProxy.info = (...args) =>{
+    log.info(...args, {requestId: namespace.get('requestId')})
+}
+logProxy.warn = (...args) =>{
+    log.warn(...args, {requestId: namespace.get('requestId')})
+}
+logProxy.error = (...args) =>{
+    log.error(...args, {requestId: namespace.get('requestId')})
+}
+logProxy.fatal = (...args) =>{
+    log.fatal(...args, {requestId: namespace.get('requestId')})
```

Comment:
Let's merge this into the object with some helper method, so it's done right, not in a `msg` as text.